src/main/java/seedu/address/logic/parser/AddTaskCommandParser.java

```diff
@@ -36,7 +36,11 @@ public AddTaskCommand parse(String args) throws ParseException {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddTaskCommand.MESSAGE_USAGE));
         }
 
-        String description = argMultimap.getValue(PREFIX_DESCRIPTION).get();
+        String description = argMultimap.getValue(PREFIX_DESCRIPTION).get().replaceAll("\\s+", " ");
```

Comment:
What does this regex do ?

Reply:
To trim whitespaces inbetween words for task description